python/ml4ir/applications/classification/pipeline.py

```diff
@@ -84,6 +84,7 @@ def get_relevance_model(self, feature_layer_keys_to_fns={}) -> RelevanceModel:
             output_name=self.args.output_name,
             logger=self.logger,
             file_io=self.file_io,
+            logs_dir=self.args.logs_dir
```

Comment:
@arvindsrikantan You should pass `logs_dir_local` here too, right?

Reply:
My bad, it has to be `logs_dir_local`